### PR TITLE
SFD-3746 Update Jenkins To Run Terraform

### PR DESCRIPTION
**Pull request by gwaardenburg**

### CODE REVIEW

File: build/jenkins/Jenkinsfile.deployment [hunk 1]
```diff
@@ -47,7 +46,6 @@ pipeline {
         }
```
Comment (daonhs) [inline build/jenkins/Jenkinsfile.deployment:46]: Doesn't look like this will run any thing

File: project.code-workspace [hunk 2]
```diff
@@ -179,27 +179,27 @@
       ]
     },
     "workbench.colorCustomizations": {
-      "activityBar.activeBackground": "#ef7853",
-      "activityBar.activeBorder": "#8af4a3",
-      "activityBar.background": "#ef7853",
+      "activityBar.activeBackground": "#82c6f3",
```
Comment (daonhs) [inline project.code-workspace:182]: Dont think you want to commit this file in

File: build/jenkins/Jenkinsfile.deployment (hunk 1 above)
Comment (gwaardenburg) [reply build/jenkins/Jenkinsfile.deployment:46]: I think this just might be github being weird. If you look at the whole file the changes seem to be there?

**Review by jope1 — DISMISSED** [review]

**Review by jope1 — APPROVED** [review]

**Review by ArifNhs — APPROVED** [review]
Approved

**Review by daonhs — APPROVED** [review]

**Pull request merged**